prophic3/array_axiom_enumerator.cpp

```diff
@@ -442,21 +401,18 @@ vector<TermSet> ArrayAxiomEnumerator::const_array_axioms_all_idx_times(Unroller
     }
     read = abstractor_.get_read(abs_ca);
 
+    msat_term lambda_j = get_finite_domain_lambda(abs_ca);
```

Comment:
do we need this for integer indices?

Reply:
No only for bit-vectors. I was thinking about going through the code and removing all the clunky bitvector stuff. But I think I’ll do that in a separate PR. Do you think that’s a good idea?

Reply:
yeah.